dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -208,6 +208,10 @@
 
     private static final String backupPath = ConfigUtils.getBackupPath() + File.separator + "contentlets";
 
+    /**
+     * Property to fetch related content from database (only applies for relationship fields)
+     * Related content for legacy relationship will always be pulled from the index
+     */
     private static final boolean getRelatedContentFromDB = Config
```

Comment:
`static final` variables should be named following the convention: `GET_RELATED_CONTENT_FROM_DB`